package.json

```diff
@@ -34,9 +34,9 @@
     "build"
   ],
   "keywords": [
```

Comment:
Not sure if we should keep the old keywords as well for search purposes, figured they'd be okay to drop, but can add them back in if needed.

Reply:
Hmm ... maybe keep the MyPureCloud since that's still the organization name for the time being.

Reply:
so, mypurecloud, genesys cloud, genesys cloud apps